src/components/Feedback/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from "react";
+import ExecutionEnvironment from "@docusaurus/ExecutionEnvironment";
+import styles from "./styles.module.css";
+
+import * as ThumbsUp from "@site/static/img/thumbs-up.svg";
+import * as ThumbsDown from "@site/static/img/thumbs-down.svg";
+
+const VotedYes = () => {
+  return (
+    <span>Thanks for your feedback! We hope this recipe has been helpful.</span>
+  );
+};
+
+const VotedNo = () => {
+  return (
+    <span>
+      Thanks for your feedback. We will update this recipe as soon as we can.
+    </span>
+  );
+};
+
+export default function Feedback({ resource }) {
+  const [reaction, setReaction] = useState(null);
```

Comment:
do we care about any type of persistence for this (local storage or beyond?)? probably not, but just adding an note.